pkg/docker/containers-view.jsx

```diff
@@ -319,7 +364,8 @@ var ImageList = React.createClass({
     getInitialState: function () {
         return {
             images: [],
-            pulling: []
+            pulling: [],
+            vulnerableInfos: {}
```

Comment:
Technically it's against our custom to have this info in the component state: This is information we'll probably also need to consume in other places (e.g. `X vulnerabilities found in Y images`). We can leave as is for now, but it'll probably need refactoring in the near future.